### PR TITLE
fix issue with sf join order

### PR DESCRIPTION
**Pull request by mountainMath**
It seems that the order in which objects are joined matters for sf objects, if the sf object is not the first argument in a join some of the spatial information will get lost, resulting in the object not rendering properly in `ggplot` without specifying a crs.

### CODE REVIEW

**Comment by dshkol** [comment]
Yeah, @atheriel, can confirm both @mountainMath and I ran into and tested out the root causes of this issue. It appears to me that the SF spatial object has to be the left hand object in dplyr::*_join functions to retain it's sf object class. Otherwise turns into a table_df class, which retains a geometry column but loses key attributes. 

What was confusing was that this issue did not previously exist based on tests on old vignettes.

File: R/cancensus.R [hunk 1]
```diff
@@ -148,8 +148,10 @@ get_census <- function (dataset, level, regions, vectors=c(), geo_format = "sf",
     result <- if (is.null(result)) {
       geos
     } else if (geo_format == "sf") {
-      dplyr::select(result, -Population, -Dwellings, -Households, -Type) %>%
-        dplyr::inner_join(geos, by = "GeoUID")
```
Comment (atheriel) [inline R/cancensus.R]: You could use `dplyr::inner_join(geos, ., by = "GeoUID") instead, if you'd like.

**Comment by atheriel** [comment]
Whoops. That's my bad.

File: R/cancensus.R (hunk 1 above)
Comment (mountainMath) [reply R/cancensus.R]: Thanks. Updated.

**Pull request merged**